mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -157,6 +157,12 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 							Required:    true,
 							Description: `The machine type to use. See tiers for more details and supported versions. Postgres supports only shared-core machine types, and custom machine types such as db-custom-2-13312. See the Custom Machine Type Documentation to learn about specifying custom machine types.`,
 						},
+						"edition": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringInSlice([]string{"EDITION_UNSPECIFIED", "ENTERPRISE", "ENTERPRISE_PLUS"}, false),
```

Comment:
`EDITION_UNSPECIFIED` can be deleted here.

We generally don't include UNSPECIFIED values for enums/strings because they're redundant / don't have a useful meaning.

Reply:
Removed `EDITION_UNSPECIFIED`.